README.md

```diff
@@ -117,10 +117,19 @@ Your app is ready to be deployed!
 Currently it includes something more than original Create React App package:
 
 * [webpack-bem-loader](https://github.com/bem/webpack-bem-loader)
+* [bem/sdk.config](https://github.com/bem/bem-sdk/tree/master/packages/config)
 * [babel-plugin-bem-import](https://github.com/bem/babel-plugin-bem-import)
 * [bem-react-core](https://github.com/bem/bem-react-core)
 * [raw-loader](https://github.com/webpack-contrib/raw-loader) includes content of *.inline.* files
 
+## Usage sets
+
+if you want to make several bundles and have their own implementation of blocks, you need to use sets
+
+1. You need specify the required sets in bem.config
+2. Then build the project. You can specify some options for [parallel-webpack](https://github.com/trivago/parallel-webpack) (watch, maxRetries, stats) Example: ```npm run build --watch=true```
+3. Finally, run app with a ```npm run start platform```
+
```

Comment:
I like shorter texts like:

* Add sets in `.bemrc`
* Build. Here you can specify some options ...
* Run with `npm start ...`